lib/commands/command-hook-asdf.bash

```diff
@@ -1,14 +1,11 @@
 #!/usr/bin/env bash
 
-use_asdf() {
-  if ! env_file="$(asdf direnv _asdf_cached_envrc "$@")"; then
-    log_error "Error generating asdf cached envrc"
```

Comment:
What happens now when we fail to generate an envrc file? Do we still get a nice error message?